content/plugins/index.md

```diff
@@ -27,16 +28,16 @@ Name                                                     | Description
 [`I18nWebpackPlugin`](/plugins/i18n-webpack-plugin)      | Add i18n support to your bundles
 [`IgnorePlugin`](/plugins/ignore-plugin)                 | Exclude certain modules from bundles
 [`LimitChunkCountPlugin`](/plugins/limit-chunk-count-plugin) | Set min/max limits for chunking to better control chunking
-|[`LoaderOptionsPlugin`](/plugins/loader-options-plugin) | Used for migrating from webpack 1 to 2
-|[`MinChunkSizePlugin`](/plugins/min-chunk-size-plugin)  | Keep chunk size above the specified limit
-|[`NormalModuleReplacementPlugin`](/plugins/normal-module-replacement-plugin) | Replace resource(s) that matches a regexp
-|[`NoEmitOnErrorsPlugin`](/plugins/no-emit-on-errors-plugin) | Skip the emitting phase when there are compilation errors
-|[`NormalModuleReplacementPlugin`](/plugins/normal-module-replacement-plugin) | Replace one resource with another
-|[`NpmInstallWebpackPlugin`](/plugins/npm-install-webpack-plugin) | Auto-install missing dependencies during development
-|[`ProvidePlugin`](/plugins/provide-plugin)              | Use modules without having to use import/require
-|[`SourceMapDevToolPlugin`](/plugins/source-map-dev-tool-plugin) | Enables a more fine grained control of source maps
-|[`UglifyjsWebpackPlugin`](/plugins/uglifyjs-webpack-plugin) | Enables control of the version of UglifyJS in your project
-|[`ZopfliWebpackPlugin`](/plugins/zopfli-webpack-plugin) | Prepare compressed versions of assets with node-zopfli
+[`LoaderOptionsPlugin`](/plugins/loader-options-plugin) | Used for migrating from webpack 1 to 2
+[`MinChunkSizePlugin`](/plugins/min-chunk-size-plugin)  | Keep chunk size above the specified limit
+[`NoEmitOnErrorsPlugin`](/plugins/no-emit-on-errors-plugin) | Skip the emitting phase when there are compilation errors
+[`NormalModuleReplacementPlugin`](/plugins/normal-module-replacement-plugin) | Replace one resource with another
+[`NormalModuleReplacementPlugin`](/plugins/normal-module-replacement-plugin) | Replace resource(s) that matches a regexp
+[`NpmInstallWebpackPlugin`](/plugins/npm-install-webpack-plugin) | Auto-install missing dependencies during development
+[`ProvidePlugin`](/plugins/provide-plugin)              | Use modules without having to use import/require
+[`SourceMapDevToolPlugin`](/plugins/source-map-dev-tool-plugin) | Enables a more fine grained control of source maps
+[`UglifyjsWebpackPlugin`](/plugins/uglifyjs-webpack-plugin) | Enables control of the version of UglifyJS in your project
+[`ZopfliWebpackPlugin`](/plugins/zopfli-webpack-plugin) | Prepare compressed versions of assets with node-zopfli
```

Comment:
Also, it looks like we have two entries for `NormalModuleReplacementPlugin`, care to fix that? The second description probably covers it fine so the first could be dropped.